### PR TITLE
Updated CONTRIBUTING.md 

### PR DESCRIPTION
**Pull request by Maghish**
Updated CONTRIBUTING.md to run newly edited back-end and front-end. Just check if the steps are correct and if this can be improved then let me know (you can simply edit it on your own as well)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,57 +13,64 @@ Welcome to Pluto! We appreciate and welcome contributions on our project, your c
...
 ```
 PORT=(The port number you want the backend to run in)
 MONGO_URI=(The uri of the MongoDB database)
 ```
 
 6. Replace the `PORT` environment variable with the port number you want the backend to run in, it is recommended to use the port number `4000` as it is the common port number used for back-end servers. For the `MONGO_URI` environment variable, you need to create an account on [MongoDB Website](https://www.mongodb.com/) and create a project, create a shared database _(the only free-tier database that MongoDB offers per project)_ and click connect button, click drivers and then choose the latest Node.js version and then copy the uri and paste it into the `MONGO_URI` environment variable.
-7. At last, you can run `npm run dev` to run the back-end server with nodemon or `npm run start` to run the back-end server with node.
+7. At last, you can run `npm run dev` to run the back-end server in nodemon.
+> [!IMPORTANT]
+> If you intend to run the back-end server in Node.js then you need to run `npm run start` but before that back-end server needs to be complied to JavaScript as it's TypeScript, so run `npm run build` and then proceed with `npm run start` to run in Node.js
```
Comment (NanotechPikachu) [inline CONTRIBUTING.md:28]: If you intend to run the back-end server in Node.js, then you need to run `npm run start`. But before that, the back-end needs to be compiled to JavaScript as it's TypeScript. So run `npm run build` and then.....

**Review by NanotechPikachu — CHANGES_REQUESTED** [review]
The review has been completed with a short note.

**Review by NanotechPikachu — APPROVED** [review]
All is well in my end

**Review by ateebplayz — APPROVED** [review]
LGTM!

**Pull request merged**